lib/openhab/dsl/monkey_patch/types/stop_move_type.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'java'
+
+module OpenHAB
+  module DSL
+    module MonkeyPatch
+      #
+      # Patches OpenHAB types
+      #
+      module Types
+        java_import Java::OrgOpenhabCoreLibraryTypes::StopMoveType
+
+        #
+        # Monkey patch for DSL use
+        #
+        class StopMoveType
+          alias inspect to_s
```

Comment:
Fair amount of duplicated code, out of curiosity did you try Monkey Patching the a common super class to alias inspect to_s? 

I can't remember how the dynamic dispatch works in JRuby and if that would work instead?

Reply:
I avoided it for now, because a) I'm not super familiar with JRuby yet, and how class hierarchies from Java come over to the Ruby object. and b) I was slightly worried about implementations for things like QuantityType/Quantity that are current sometimes-wrapped from their OH object. I'm happy to look into it more.